### ISSUE TITLE
HelloMvc Ubuntu 

### ISSUE THREAD

**Issue by cataquil** [aspnet/Home#1227]
Hello I followed the setup.  The console app works, HelloWeb works

this is using the "dnx web" for HelloWeb.

When I try this on HelloMvc I get 

'Microsoft.AspNet.Server.Kestrel' does not contain a static 'Main' method suitable for an entry point

I don't understand because the project.json is the same.

So I googled around, some blogs several months old, things are in beta...where are we now. 

I tried 'dnx HelloMvc'  then I get

Error: /home/userName/Home/samples/latest/HelloMvc/Startup.cs(16,27): DNX,Version=v4.5.1 error CS1061: 'ILoggerFactory' does not contain a definition for 'AddConsole' and no extension method 'AddConsole' accepting a first argument of type 'ILoggerFactory' could be found (are you missing a using directive or an assembly reference?)

So it is probably a dependency, using the wrong nuget, something like that.

This is the output of 'dnvm list'

Active Version              Runtime Architecture OperatingSystem Alias

---

```
   1.0.0-rc1-update1    coreclr x64          linux           
```
-    1.0.0-rc1-update1    mono                 linux/osx       default

**Comment by cataquil** [comment]
Have to roll back mono.
http://www.shrayas.com/asp-net-5-npgsql-linux-mono-4-2.html

editing /etc/apt/sources.list.d was a pain
used the following on lubuntu
gksudo leafpad  mono-xamarin.list

**Comment by sec** [comment]
Try fix from #1093

**Comment by aspnet-hello** [comment]
This issue is being closed because it has not been updated in 3 months.

We apologize if this causes any inconvenience. We ask that if you are still encountering this issue, please log a new issue with updated information and we will investigate.
